sympy/printing/mathml.py

```diff
@@ -861,15 +865,29 @@ def _print_binomial(self, expr, exp=None):
 
     def _print_Pow(self, e):
         # Here we use root instead of power if the exponent is the reciprocal of an integer
-        if e.exp.is_negative or len(str(e.base)) > 1:
-            mrow = self.dom.createElement('mrow')
-            x = self.dom.createElement('mfenced')
-            x.appendChild(self._print(e.base))
-            mrow.appendChild(x)
-            x = self.dom.createElement('msup')
-            x.appendChild(mrow)
-            x.appendChild(self._print(e.exp))
-            return x
+        if e.exp.is_negative:
+            if self._settings['fold_short_frac'] and len(str(e.base)) < 3:
```

Comment:
Can you please provide an example of MathML code generated from this part of the code? I cannot really figure out what it looks like (and when this happens).

Reply:
```
In [3]: a= 1/(x)                                                                                                                                                                                                   

In [4]: print_mathml(a, printer='presentation', fold_short_frac=True)                                                                                                                                              
shubham
<mfrac bevelled="true">
	<mn>1</mn>
	<msup>
		<mrow>
			<mfenced>
				<mi>x</mi>
			</mfenced>
		</mrow>
		<mn>1</mn>
	</msup>
</mfrac> 
```
Is this how you want?

Reply:
Thanks! 

I am not sure if this should happen or not. Well, this particular example shouldn't happen, but in general.

Here, you will need to check if the exponent is one and in that case not print the exponent, now it looks like
![image](https://user-images.githubusercontent.com/8114497/53441688-a17cbf00-3a07-11e9-8c80-42c822092b2a.png)

There is a method in another PR to check if there should be brackets around the expression, now maybe one could check if it `e.base` is `Symbol` and if it is not print the bracket.

Reply:
Which PR is that to check for ```e.base``` is ```Symbol``` ? or to check for brackets?

Reply:
The bracket check is actually merged. I didn't realize that (I have two outstanding PRs on this and I though it was in one of those). See #16069 

I have also realized that the behavior of the MathML Rational is completely different to the LaTeX one, so as long as the 1-exponent is not printed I think this would be good to go for now. One can always come back and make it more like the LaTeX one if that is preferred.